SofaKernel/modules/SofaCore/src/sofa/core/fwd.cpp

```diff
@@ -23,8 +23,8 @@ const ConstraintParams* defaultInstance()
     return ConstraintParams::defaultInstance();
 }
 
-ExecParams* asExecParams(sofa::core::ConstraintParams* cparams){ return cparams; }
-const ExecParams* asExecParams(const sofa::core::ConstraintParams* cparams){ return cparams; }
+ExecParams* dynamicCastToExecParams(sofa::core::ConstraintParams* cparams){ return cparams; }
+const ExecParams* dynamicCastToExecParams(const sofa::core::ConstraintParams* cparams){ return cparams; }
```

Comment:
dynamicCast function without using `dynamic_cast` ?

Reply:
lol you are right. As it is a direct parent there is no need for a cast here.

Reply:
So I finally swtich to the following convention:
castTo<AnyType>(Base*)
castToBase(aSubTypeOfBase)
castToExecParams(ASubTypeOfExecParam)...